entrypoint.sh

```diff
@@ -38,6 +38,10 @@ GITHUB_TOKEN=${INPUT_GITHUB_TOKEN:-$GITHUB_TOKEN}
 # Specifies the Git remote repository
 # REMOTE_REPO=${REMOTE_REPO:-"https://${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git"}
 REMOTE_REPO=${INPUT_REMOTE_REPO:-${REMOTE_REPO:-"https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git"}}
+
+# Specifies the Jekyll build command
+JEKYLL_BUILD_OPTS="${INPUT_JEKYLL_BUILD_OPTS:-${JEKYLL_BUILD_OPTS}}"
```

Comment:
Could you move this variable declaration below the Jekyll-related options such that it's not out-of-place among the other variable declarations?

Reply:
Done in 7615fc5f6d1c0a69bb3f6f7b7bbbc4f312d82b11.